app/build.gradle

```diff
@@ -2,6 +2,7 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
+apply plugin: 'androidx.navigation.safeargs.kotlin'
```

Comment:
I wanted this to be alphabetical but it needs to come after the `com.android.application` plugin :|